docs: Consolidate pages for access user management (#168)
langchain-ai/docs: src/langsmith/billing.mdx

Personal organizations are limited to 5000 traces per month until a credit card is added. You can add a credit card on the Plans and Billing page as follows:

1. Click **Set up Billing**

  ![](/langsmith/images/free-tier-billing-page.png)

2. Add your credit card information. After this step, you will no longer be rate limited to 5000 traces, and you will be charged for any excess traces at rates specified on the [pricing](https://www.langchain.com/pricing-langsmith) page.

### Plus Plan: set up billing on a shared organization

If you have not yet created an organization, you need to follow [this guide](/langsmith/set-up-a-workspace#set-up-an-organization) before setting up billing. The following steps assume you are already in a new organization.

<Note>
You cannot use a new organization until you have entered credit card information. After you complete the following steps, you will gain complete access to LangSmith.
</Note>

1. Click **Subscribe** on the **Plus** page.

  <Note>
  If you are a startup building with AI, instead click **Apply Now** on the Startup Plan. You may be eligible for discounted prices and a free, monthly trace allotment.
  </Note>

>   ![](/langsmith/images/new-org-billing-page.png)

Author: Yes, we should! We have an issue open to tackle this post migration.

To set limits, navigate back to **Settings** -> **Usage and Billing** -> **Usage configuration**. There is a table at the bottom of the page that lets you set usage limits per workspace. For each workspace, the two limits appear, along with a cost estimate:

![](/langsmith/images/p2usagelimitsempty-v2.png)

Start by setting limits on production usage, since that is where the majority of spend comes from.

#### Set a good total traces limit

Picking the right "total traces" limit depends on the expected load of traces that you will send to LangSmith. It is important to consider potential growth before setting a limit. For example:

- **Current Load**: The gen AI application is called between 1.2-1.5 times per second, and each API request has a trace associated with it, meaning it logs around 100,000-130,000 traces per day.
- **Expected Growth in Load**: The expectation is that this will double in size in the near future.

From these assumptions, you can calculate an approximate limit:

```python
limit = current_load_per_day * expected_growth * days/month
      = 130,000 * 2 * 30
      = 7,800,000 traces / month
```

> Click on the edit icon on the right side of the table for our **Prod** row, you can enter this limit as follows:

Author: ```suggestion
Click on the edit icon on the right side of the table for the **Prod** row to enter the limit.
```

![](/langsmith/images/p2alllimitonly-v2.png)

<Note>
When set without the extended data retention traces limit, the maximum cost estimator assumes that all traces are using extended data retention.
</Note>

#### Cut maximum spend with an extended data retention limit

From [Optimization 1](#optimization-1-manage-data-retention), you learned that the easiest way to cut cost was through managing data retention. The same is true for limits. If you only want to keep roughly 10% of traces to be around more than 14 days, you can set a limit on the maximum high retention traces you can keep. This would result in `.10 * 7,800,000 = 780,000`.

![](/langsmith/images/p2bothlimits-v2.png)

> The maximum cost is cut from \~40k per month to \~7.5k per month, because you no longer allow as many expensive data retention upgrades. This ensures that new users on the platform will not accidentally cause cost to balloon.

Author: yes, it seems like they were needed as per the old content

<Note>

> The extended data retention limit can cause features other than traces to stop working once reached. If you plan to use this feature, please read more about its functionality [here](/langsmith/administration-overview#side-effects-of-extended-data-retention-traces-limit).

Reviewer: ```suggestion
The extended data retention limit can cause features other than traces to stop working once reached. If you plan to use this feature, read more about its [functionality and side effects](/langsmith/administration-overview#side-effects-of-extended-data-retention-traces-limit).
```
